mitosheet/src/mito/components/toolbar/HomeTabContents.tsx

```diff
@@ -146,6 +146,10 @@ export const HomeTabContents = (
                         }}/>
                 </Dropdown>
             </ToolbarButton>
+            <ToolbarButton
```

Comment:
Nice, this file is very easy to edit, and the resulting file looks super clear + clean + easy to maintain going forward!